src/api/export.ts

```diff
@@ -115,19 +130,26 @@ async function _export({
 			if (pathname !== '/service-worker-index.html') {
 				file = file === '' ? 'index.html' : `${file}/index.html`;
 			}
-			body = minify_html(body);
+
+			if(typeof(body)!=='string'){
```

Comment:
nit: should add a spaces and I don't think you need the parens:

    if (typeof body !== 'string') {